openlibrary/plugins/openlibrary/home.py

```diff
@@ -51,17 +52,20 @@ def get_homepage():
     )
     return dict(page)
 
-def get_cached_homepage():
+
+def get_cached_homepage(lang='en'):
```

Comment:
This no longer needs to be optional.

```suggestion
def get_cached_homepage(lang):
```

Reply:
Done